src/core/Table/hooks/useSelectionCell.tsx

```diff
@@ -26,6 +26,7 @@ export const useSelectionCell = <T extends Record<string, unknown>>(
       maxWidth: 48,
       columnClassName: 'iui-slot',
       cellClassName: 'iui-slot',
+      disableReordering: true,
```

Comment:
Is there are difference between `disableReordering` and `enableReordering` prop names? 
Why in table we use positive version `enableColumnReordering` and here negative `disable...`?